docs/admin/scale/index.md

```diff
@@ -0,0 +1,69 @@
+We scale-test Coder with the [same utility](#scaletest-utility) that can be used in your environment for insights into how Coder scales with your infrastructure.
+
+## General concepts
+
+Coder runs workspace operations in a queue. The number of concurrent builds will be limited to the number of provisioner daemons across all coderd replicas.
+
+- **coderd**: Coder’s primary service. Learn more about [Coder’s architecture](../../about/architecture.md)
+- **coderd replicas**: Replicas (often via Kubernetes) for high availability, this is an [enterprise feature](../../enterprise.md)
+- **concurrent workspace builds**: Workspace operations (e.g. create/stop/delete/apply) across all users
+- **concurrent connections**: Any connection to a workspace (e.g. SSH, web terminal, `coder_app`)
+- **provisioner daemons**: Coder runs one workspace build per provisioner daemon. One coderd replica can host many daemons
+- **scaletest**: Our scale-testing utility, built into the `coder` command line.
+
+```text
+2 coderd replicas * 30 provisioner daemons = 60 max concurrent workspace builds
+```
+
+## Infrastructure recommendations
+
+### Concurrent workspace builds
+
+Workspace builds are CPU-intensive, as it relies on Terraform. Various [Terraform providers](https://registry.terraform.io/browse/providers) have different resource requirements. When tested with our [kubernetes](https://github.com/coder/coder/tree/main/examples/templates/kubernetes) template, `coderd` will consume roughly 8 cores per 30 concurrent workspace builds. For effective provisioning, our helm chart prefers to schedule [one coderd replica per-node](https://github.com/coder/coder/blob/main/helm/values.yaml#L110-L121).
+
+To support 120 concurrent workspace builds, for example:
+
+- Create a cluster/nodepool with four 8-core nodes (AWS: `t3.2xlarge` GCP: `e2-highcpu-8`)
+- Run coderd with 4 replicas, 30 provisioner daemons each. (`CODER_PROVISIONER_DAEMONS=30`)
+- Ensure Coder's [PostgreSQL server](../../admin/configure.md#postgresql-database) can use up to 1.5 cores
+
+## Recent scale tests
+
+| Environment        | Users | Concurrent builds | Concurrent connections (SSH) | Concurrent connections (web) | Last tested  |
+| ------------------ | ----- | ----------------- | ---------------------------- | ---------------------------- | ------------ |
```

Comment:
It might be relevant for users to know what's the total time of the scale test. If scale tests performed on their infra take twice as much time, it will be a hint for them to start digging.

Reply:
We can make sure to include the time next time we run these tests.

Reply:
You may want to add a column already, so the next person will not forget about measuring it.